awx/main/tests/unit/models/test_label.py

```diff
@@ -37,39 +45,101 @@ def test_is_detached_not(self, mocker):
         ret = label.is_detached()
 
         assert ret is False
-        Label.objects.filter.assert_called_with(id=37, unifiedjob_labels__isnull=True, unifiedjobtemplate_labels__isnull=True, inventory_labels__isnull=True)
+        Label.objects.filter.assert_called_with(
+            id=37,
+            unifiedjob_labels__isnull=True,
+            unifiedjobtemplate_labels__isnull=True,
+            inventory_labels__isnull=True,
+            schedule_labels__isnull=True,
+            workflowjobtemplatenode_labels__isnull=True,
+            workflowjobnode_labels=True,
+        )
+
         mock_query_set.exists.assert_called_with()
 
     @pytest.mark.parametrize(
-        "jt_count,j_count,inv_count,expected",
+        "jt_count,j_count,inv_count,sched_count,wfnode_count,wfnodej_count,expected",
         [
-            (1, 0, 0, True),
-            (0, 1, 0, True),
-            (0, 0, 1, True),
-            (1, 1, 1, False),
+            (1, 0, 0, 0, 0, 0, True),
+            (0, 1, 0, 0, 0, 0, True),
+            (1, 1, 0, 0, 0, 0, False),
```

Comment:
It might be easier to read if you reordered these, so that the 1s for the `True` cases go in a straight line from the upper left to bottom right, like `\`

Reply:
They are in that order because I treated the 1's and 0's like a 6 digit binary number to make sure I didn't miss something.